uwblueprint/ccbc
# [Feat] Search and Filter Base Review Page

## Notion ticket link

[Search and Filter Base Review Page](https://www.notion.so/uwblueprintexecs/16fb020b93f641c8b384f76a1b3c769e?v=aedc8d99cd8b437cb556ece10f3c0e3c&p=169bc7491da340a9876892d52841247f)



## Implementation description
* Store state of url params and search query in search bar
* Put review grid and search bar together
* OMIT: filters component because not ready yet -> new ticket here: https://www.notion.so/uwblueprintexecs/16fb020b93f641c8b384f76a1b3c769e?v=aedc8d99cd8b437cb556ece10f3c0e3c&p=db301040f994483388abec0906627633&pm=c



## Steps to test
1. Go to `http://localhost:3000/magazine/search_results`
2. search something in the search bar and verify that it updates the url
3. http://localhost:3000/magazine/search_results?search_query=RANDOM TEXT HERE` and verify that the searchbar is updated
4. verify that page is responsive by resizing



## What should reviewers focus on?
* 


## Checklist
